ajax/libs/jqgrid/package.json

```
@@ -1,9 +1,8 @@
 {
   "name": "jqgrid",
-  "filename": "js/jquery.jqGrid.min.js",
-  "version": "4.6.0",
+  "filename": "js/jquery.jqGrid.src.js",
```

```suggestion
  "filename": "js/jquery.jqGrid.min.js",
```
The minified file appears to be what is included in the repo and we should always prefer to recommend a minified file where available: https://github.com/tonytomov/jqGrid/tree/v5.4.0/js